### PR TITLE
CC-1269: Backport clojure deps update to old versions

### PR DESCRIPTION
**Pull request by ryan-gang**

### CODE REVIEW

**Comment by ryan-gang** [comment]
Updated codecrafters.yml temporarily, will revert before merging.

File: compiled_starters/clojure/codecrafters.yml [hunk 1]
```diff
@@ -7,5 +7,5 @@ debug: false
 # Use this to change the Clojure version used to run your code
 # on Codecrafters.
 #
-# Available versions: clojure-1.11.3
-language_pack: clojure-1.11.3
+# Available versions: clojure-1.10.3
+language_pack: clojure-1.10.3
```
Comment (rohitpaulk) [inline compiled_starters/clojure/codecrafters.yml:11]: Not sure what's happening here - we're changing the default to an older version, and we're renaming `clojure-1.11.3.Dockerfile` to `clojure-0.Dockerfile`?

**Review by rohitpaulk — CHANGES_REQUESTED** [review]
Added comment

File: compiled_starters/clojure/codecrafters.yml (hunk 1 above)
Comment (rohitpaulk) [reply compiled_starters/clojure/codecrafters.yml:11]: Ah, ignore - just saw your comment

**Review by rohitpaulk — APPROVED** [review]

**Pull request merged**